packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -115,8 +118,11 @@ export class Checkbox extends BaseComponent<ICheckboxProps, ICheckboxState> impl
         onBlur={ this._onBlur }
         aria-checked={ isChecked }
         aria-disabled={ disabled }
+        aria-label={ ariaLabel }
+        aria-labelledby={ ariaLabelledBy }
+        aria-describedby={ ariaDescribedBy }
       >
-        <label className={ this._classNames.label } htmlFor={ this._id } >
+        <label className={ this._classNames.label } htmlFor={ ariaLabelledBy || ariaLabel ? undefined : this._id } >
```

Comment:
I'm a bit unsure what the intent is here.

As a customer, if I provide `ariaLabelledBy` pointing to my customized label element, why do i want the Checkbox's label element to have htmlFor pointing to that customized label element? Is this correct?